addon/controllers/admin/model-records/show/related/index.js

```diff
@@ -14,33 +14,28 @@ export default Controller.extend(
   {
     actions: {
       remove(model, parentModel) {
-        const recordName = this.get('recordName')
-        const inverseRelationshipName =
-          this.inverseRelationshipName(parentModel, recordName)
-        const kind =
-          this.relationshipKind(parentModel, recordName)
-        const inverseKind =
-          this.inverseRelationshipKind(parentModel, recordName)
+        const {
+          inverseRelationshipKind,
+          inverseRelationshipName,
+          relationshipKind,
+          relationshipName
+        } = this.relationshipInfo(parentModel)
 
-        if (kind === 'belongsTo') {
-          parentModel.set(recordName, null)
-        } else {
-          parentModel.get(recordName).removeObject(model)
-        }
+        parentModel
+          .removeRelated(relationshipName, model, relationshipKind)
+          .then(() => {
```

Comment:
Is this still necessary?   It seems like the response from the above ajax call should handle syncing the changes, or if needs extra munging, handled in the removeRelated method.